data/tribes/initialization/frisians/units.lua

```diff
@@ -894,183 +894,253 @@ tribes:new_tribe {
       {
          name = "berry_bush_blueberry_tiny",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush has just been planted.")
          }
       },
       {
          name = "berry_bush_blueberry_small",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
          }
       },
       {
          name = "berry_bush_blueberry_medium",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
          }
       },
       {
          name = "berry_bush_blueberry_ripe",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is ready for harvesting.")
          }
       },
       {
          name = "berry_bush_currant_red_tiny",
          helptexts = {
-
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush has just been planted.")
          }
       },
       {
          name = "berry_bush_currant_red_small",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
          }
       },
       {
          name = "berry_bush_currant_red_medium",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is growing.")
          }
       },
       {
          name = "berry_bush_currant_red_ripe",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush is ready for harvesting.")
          }
       },
       {
          name = "berry_bush_juniper_tiny",
          helptexts = {
+            -- TRANSLATORS: Helptext for a frisian immovable: One of the berry bushes
+            purpose = _("This berry bush has just been planted.")
```

Comment:
Juniper is not a berry, please use just "This bush …" for this species

Reply:
Actually, juniper is not a bush, it's a shrub. And then we have to explain that the fruit collector collects fruit from bushes and shrubs, and not from "berry bushes" etc. I'd say let's simply use game terminology here.